gnoboard/src/hooks/use-gno.ts

```diff
@@ -33,12 +31,11 @@ interface GnoResponse {
   createAccount: (nameOrBech32: string, mnemonic: string, password: string) => Promise<GnoAccount | undefined>;
   generateRecoveryPhrase: () => Promise<string>;
   listKeyInfo: () => Promise<GnoAccount[]>;
+  isAccountNameExists: (name: string) => Promise<boolean>;
```

Comment:
I think Remi made the same comment. Why do you need isAccountNameExists. This is the same as hasKeyByName.

Reply:
Old code. Fixed with a rebase.